app/assets/stylesheets/opportunities.scss

```diff
@@ -23,6 +23,13 @@ h1, h2 {
   color: white;
 }
 
+.at-capacity {
+  background-color: #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`